### PR TITLE
Fix type of timestamp in monitor

### LINKED ISSUE #1921: QCoDeS monitor broken on master
The qcodes monitor is broken on the latest master. The error is occuring somewhere in the javascript as the data is still correctly sent over the websocket.

### Steps to reproduce
1. Create a monitor in QCoDeS.
2. Launch the monitor using `python -m qcodes.monitor.monitor`

### Expected behaviour
QCoDeS monitor displays in webbrowser.

### Actual behaviour
I get an error message in both Firefox and IE. (I don't have a copy of chrome to test on my experimental PC.

The error is: 
```
Status:error:I ran into the following problems at _.parameters[0].parameters[34].ts: Expecting null but instead got: "1581406601.743032" Expecting a Float but instead got: "1581406601.743032"
```

The issue seems to have crept in between revisions e5c5324 and 1ce7a23.

I've attached a sample response from the websocket in the working state and the broken state, where you can see that the timestamp is being sent as a string instead of a float...

[websocket_brokensample.txt](https://github.com/QCoDeS/Qcodes/files/4184596/websocket_brokensample.txt)
[websocket_workingsample.txt](https://github.com/QCoDeS/Qcodes/files/4184597/websocket_workingsample.txt)

### System
Windows 10
QCodes Master 33d4e71

### PR DESCRIPTION
**Pull request by spauka**
Fixes #1921 .

Changes proposed in this pull request:
- Corrects type of timestamp in the monitor

This could also be fixed in the web frontend code, but I can't seem to find that anywhere...

@jenshnielsen

### CODE REVIEW

**Comment by jenshnielsen** [comment]
Thanks @spauka Just left one question about the changes to socket server.

File: qcodes/monitor/monitor.py [hunk 1]
```diff
@@ -267,7 +266,7 @@ def show() -> None:
     os.chdir(STATIC_DIR)
     try:
         log.info("Starting HTTP Server at http://localhost:%i", SERVER_PORT)
-        with socketserver.TCPServer(("", SERVER_PORT),
```
Comment (jenshnielsen) [inline qcodes/monitor/monitor.py]: I don't think this change is needed. There is a bug in typing shed with mypy 0.761 that breaks the types around here. I have submitted a pr for that here so it should be fixed in the next version. https://github.com/python/typeshed/pull/3579/files this is why qcodes is checked with mypy 0.750 btw

Comment (spauka) [reply qcodes/monitor/monitor.py]: Haha sounds good 😄 do you think it's better to revert this change? I am happy for you to do that, or I can too :P

Comment (jenshnielsen) [reply qcodes/monitor/monitor.py]: Yes if you can confirm that you only made the change to keep mypy happy lets revert it. I am happy to do so

Comment (spauka) [reply qcodes/monitor/monitor.py]: Awesome, I just left work so feel free :+1:

**Pull request merged**